tests/Unit/ContactTest.php

```diff
@@ -661,16 +697,14 @@ public function test_it_returns_an_unknown_birthday_state()
 
     public function test_it_returns_an_approximate_birthday_state()
     {
-        $contact = factory(Contact::class)->create([
-            'account_id' => 1,
-        ]);
+        $contact = factory(Contact::class)->create();
         $specialDate = factory(SpecialDate::class)->create([
             'is_age_based' => 1,
         ]);
         $contact->birthday_special_date_id = $specialDate->id;
         $contact->save();
 
-        $specialDate->contact_id = $specialDate->id;
+        $specialDate->contact_id = $contact->id;
```

Comment:
Ahah good catch. How come this wasn't triggering an error?

Reply:
It's an `id`, so it's technically valid :laughing:.

Reply:
One more good one.